rise-storage.html

```diff
@@ -643,46 +643,14 @@
     /**
      * Processes a Storage file.
      */
-    _handleStorageFile: function(resp) {
-      var file = {},
-        etag = null;
-
-      // File in the root of the bucket.
-      if (resp.selfLink !== undefined) {
-        etag = resp.etag;
-      }
-      // File in a folder.
-      else if ((resp.files !== undefined) && (resp.files.length > 0)) {
-        etag = resp.files[0].etag;
-      }
+    _handleStorageFile: function() {
+      var file = {};
 
       file.url = this._fileUrl;
-
-      if (this._isLoading) {
-        this._files.push({
-          "name": this._getFileNameFromUrl(file.url),
-          "etag": etag
-        });
-
-        file.added = true;
-
-        this._isLoading = false;
-      }
-      else {
-        // File hasn't changed.
-        if (this._files[0].etag === etag) {
-          file.changed = false;
-        }
-        else {
-          this._files[0].etag = etag;
-          file.changed = true;
-        }
-      }
-
+      file.changed = true;
```

Comment:
@rodrigopavezi I guess I'm a bit confused because `_handleStorageFile` is a function that only gets called when RC is not running and when RC is not running there's no refresh. So I'm not seeing a scenario where "rise-storage-response" is fired with `file.changed`. If you think this is more robust though then I'm good with that. Cheers